arcadebackend/index.js

```diff
@@ -1,20 +1,23 @@
 import express, {json} from "express"
 import { exec } from "child_process"
+import cors from "cors"
+import path from "path"
 
 const app = express()
 
-const PORT = 3000
+const PORT = 3001
```

Comment:
we might need to change the ports because the backend uses the ports from 3000 to 3002. we can fix this in staging.